sdk/aqueduct/integrations/sql_integration.py

```diff
@@ -54,6 +47,28 @@ def replace_today() -> str:
     return "'" + date.today().strftime("%Y-%m-%d") + "'"
 
 
+def find_parameter_names(query: str) -> List[str]:
+    matches = re.findall(TAG_PATTERN, query)
+    return [match.strip(" {}") for match in matches]
+
+
+def find_parameter_artifacts(dag: DAG, names: List[str]) -> List[ArtifactMetadata]:
```

Comment:
The semantics are a little unexpected here, since `len(names) != len(returned artifacts)` in some cases. Can you put a comment here that not all names will have a corresponding artifact, and why that would be the case?